README.rst

```diff
@@ -23,6 +23,23 @@ The contents will be:
 - power colours
 - rms spectra
 
+Installation
+-------------
```

Comment:
Argh, I've never heard about it sorry.
Hm... what is a purpose of installing dependencies via pip - it should be installed via
`python setup.py install`

One thing for contributors - might be good to mention about develop command
`python setup.py develop`